Composer/packages/client/src/pages/home/CardWidget.tsx

```diff
@@ -68,32 +63,41 @@ export const CardWidget: React.FC<CardWidgetProps> = ({
     }
   };
 
+  const labelId = useId('link-label');
+
   return (
-    <a
-      css={[itemContainerWrapper(disabled), styles.container]}
-      href={href}
-      target={target}
-      onClick={async (e) => {
-        if (onClick != null) {
-          e.preventDefault();
-          await onClick();
-        }
-      }}
-      {...rest}
```

Comment:
is there anything in `rest` that needs to be spread onto the anchor tag or is it ok to remove this?

Reply:
There is no anything to spread to anchor. The new `role` prop should go directly to the root

Reply:
Looks good then!